controllers/apply/digital-funding-demo/form-model.js

```diff
@@ -106,12 +106,11 @@ module.exports = strandNumber => {
         title: `Digital Funding Strand ${strandNumber} (Demo)`,
         pageAccent: 'blue',
         heroSlug: 'whizz-kidz',
-        hidePagination: true,
         shortCode: `DF${strandNumber}-ALPHA`,
         steps: strandSteps[`strand${strandNumber}`],
         processor: processor,
         startPage: {
-            template: path.resolve(__dirname, './startpage')
+            path: `/funding/programmes/digital-funding-demo/strand-${strandNumber}`
```

Comment:
As this is the one part of "real" code we're touching, could we name this more explicitly, maybe `urlPath`. The `template` property is a path too, albeit a file path, but still keen to make the distinction clear.

Reply:
Good call – done.